Solutions/Menes.Specs/Menes.Specs.csproj

```diff
@@ -63,27 +63,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Update="Features\HttpResultBuilderErrorDetection.feature.cs">
-      <DependentUpon>HttpResultBuilderErrorDetection.feature</DependentUpon>
-    </Compile>
-    <Compile Update="Features\OpenApiDefaultParameterParsing.feature.cs">
-      <DependentUpon>OpenApiDefaultParameterParsing.feature</DependentUpon>
-    </Compile>
-  </ItemGroup>
   <ItemGroup>
     <None Update="specflow.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  <ItemGroup>
-    <SpecFlowFeatureFiles Update="Features\HttpResultBuilderErrorDetection.feature">
-      <Visible>$(UsingMicrosoftNETSdk)</Visible>
-      <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
-    </SpecFlowFeatureFiles>
-    <SpecFlowFeatureFiles Update="Features\OpenApiDefaultParameterParsing.feature">
-      <Visible>$(UsingMicrosoftNETSdk)</Visible>
-      <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
-    </SpecFlowFeatureFiles>
-  </ItemGroup>
 </Project>
```

Comment:
I'm guessing SpecFlow has just splurged all this stuff into the csproj

Reply:
Yes, it keeps doing that at random. As far as I can tell, ever since they got their build integration sorted, none of this stuff needs to be in your own project file. (Because the SpecFlow NuGet packages are able to inject their own target files into the build, there's no need for edits to our own project files.)

I _think_ the reason this happens is that the SpecFlow extension sometimes mistakenly things it needs to add it. I don't really know why though. No harm ever seems to come from deleting all this stuff though.,